assets/js/main.js

```diff
@@ -165,6 +166,36 @@ $(document).ready(function () {
         }, 1000)
     }
 
+    async function setAnalyticsEvents() {
+        await sendEvent('page_viewed')
+
+        $(document).bind('copy', async function () {
```

Comment:
In the future, avoid adding new jQuery functions, use vanilla js. That way there's less code to refactor when we start moving away from jQuery in docs as well.